docs/DeveloperGuide.md

```diff
@@ -2,36 +2,118 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}  
+Zhou Zhou's Individual Project: https://github.com/maanyos/ip \
+Nay Chi's Individual Project: https://github.com/NayChi-7/ip
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Class Diagrams
+
+Biometrics component:  
+The Biometrics component stores information about the user's biometrics and holds a reference
+to an ArrayList of the user's weight and fat records under WeightAndFatList class.  
+WeightAndFatList is separated from Biometrics for better abstraction and cohesion.  
+[Proposed] Weight and fat records will be associated with a date, and the WeightAndFatList class
+stores records in descending order of date for viewing purposes.
+![Biometrics.png](UmlDiagrams/Biometrics.png)
```

Comment:
maybe can make some of the boxes wider so that each method is in one line instead of continuing on in the second line
for example: the `addWeightAndFat` method
![image](https://user-images.githubusercontent.com/88262276/198495836-6cd8c433-1b5c-47d8-9e10-f03104e38b9c.png)